assets/app/widgets/suse-lazy-spinner/suse-lazy-spinner.directive.js

```diff
@@ -0,0 +1,58 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('crowbarWidgets')
+        .directive('suseLazySpinner', lazySpinner);
```

Comment:
To keep consistency, I'd use the same name for both directive's and function's name